tests/collections/asr/test_speaker_label_models.py

```diff
@@ -61,3 +61,34 @@ def test_constructor(self):
         confdict = speaker_model.to_config_dict()
         instance2 = EncDecSpeakerLabelModel.from_config_dict(confdict)
         self.assertTrue(isinstance(instance2, EncDecSpeakerLabelModel))
+
+    @pytest.mark.unit
+    def test_ecapa_enc_dec(self):
+        preprocessor = {'cls': 'nemo.collections.asr.modules.AudioToMelSpectrogramPreprocessor', 'params': dict({})}
+        encoder = {
+            'cls': 'nemo.collections.asr.modules.ECAPA_Encoder',
+            'params': {
+                'feat_in': 80,
+                'filters': [1024, 1024, 1024, 1024, 3072],
```

Comment:
Dont use such large filters for tests, it will slow it down.

Reply:
you are right